app/core/components/OnboardingQuests.jsx

```diff
@@ -1,14 +1,23 @@
 import { useMutation, useQuery } from "blitz"
 import { Widget } from "@uploadcare/react-widget"
-import { useRef } from "react"
+import { useEffect, useRef, useState } from "react"
```

Comment:
Do we need to load these dependencies? If not, maybe roll back to like this?
```suggestion
import { useRef } from "react"
```